notify/notify.sh

```diff
@@ -3,17 +3,6 @@
 pname notify/notify
 
 buttonset() {
-    if [ -n "$2" ]; then
-        if [ "$2" = "0" ]; then
-            BUTTONSTATE=false
-        else
-            if [ "$2" = "1" ]; then
-                BUTTONSTATE=true
-            fi
-        fi
-    else
-        BUTTONSTATE=false
-    fi
-
-    notify-send.py a --hint boolean:deadd-notification-center:true int:id:"$1" boolean:state:"$BUTTONSTATE"
+    [ "$2" = "1" ] && BUTTONSTATE=true
+    notify-send.py a --hint boolean:dead-notification-center:true int:id:"$1" boolean:state:"${BUTTONSTATE:-false}"
```

Comment:
Amazing how much you can refine code. xD I was unable to test this as neither your nor my code works with my `notify-send`, and I don't have whatever `notify-send.py` is. Should work okay though. Just improved logic, really.

Reply:
[notify-send.py](https://github.com/phuhl/notify-send.py) is an improved version of notify-send. It belongs to the [notification center](https://github.com/phuhl/linux_notification_center) I use for my [suckless](https://paperbenni.github.io/suckless) build

Reply:
Oh, that's cool. Did my code change work with it?